outputs/docker.go

```diff
@@ -0,0 +1,108 @@
+package outputs
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"log"
+
+	"github.com/aquasecurity/postee/v2/layout"
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/api/types/container"
+	"github.com/docker/docker/api/types/mount"
+	"github.com/docker/docker/client"
+	"github.com/docker/docker/pkg/stdcopy"
+	"github.com/google/uuid"
+)
+
+type DockerClient struct {
+	client  client.APIClient
+	uuidNew func() uuid.UUID
+
+	Name      string
+	ImageName string
+	Cmd       []string
+	Volumes   map[string]string
+	Env       []string
+}
+
+func (d DockerClient) GetName() string {
+	return d.Name
+}
+
+func (d *DockerClient) Init() error {
+	var err error
+	d.client, err = client.NewClientWithOpts(client.FromEnv, client.WithAPIVersionNegotiation())
+	if err != nil {
+		return fmt.Errorf("failed to initialize docker action: %w", err)
+	}
+	d.uuidNew = uuid.New
+
+	log.Println("docker action successfully initialized")
+	return nil
+}
+
+func (d DockerClient) Send(m map[string]string) error {
+	ctx := context.Background()
+
+	_, err := d.client.ImagePull(ctx, d.ImageName, types.ImagePullOptions{})
+	if err != nil {
+		return fmt.Errorf("docker action failed to pull docker image: %w", err)
+	}
+
+	var hc container.HostConfig
+	if len(d.Volumes) > 0 {
+		for src, dst := range d.Volumes {
+			hc.Mounts = append(hc.Mounts, mount.Mount{Type: mount.TypeBind, Source: src, Target: dst})
+		}
+	}
+	d.Env = append(d.Env, fmt.Sprintf("POSTEE_EVENT=%s", m["description"]))
```

Comment:
I didn't find where it's cleared. It may be required if action is invoked more than once

Reply:
thanks - fixed https://github.com/aquasecurity/postee/pull/312/commits/e39af060ab40098ce57ee55460ab8504045acbb1